<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>全场定位上位机</string>
  </property>
  <property name="locale">
   <locale language="Chinese" country="China"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="serialStatus">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>未连接串口</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>30</y>
      <width>413</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="1">
      <widget class="QComboBox" name="serialBox"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>选择串口</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="refreshSerialButton">
       <property name="text">
        <string>刷新串口</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="switchSerialButton">
       <property name="text">
        <string>打开串口</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="logoLable">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="dataGroup">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>721</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="currentGroup">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>342</width>
       <height>259</height>
      </rect>
     </property>
     <property name="title">
      <string>当前数据</string>
     </property>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>301</width>
        <height>221</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0" colspan="3">
        <layout class="QGridLayout" name="gridLayout">
         <item row="4" column="1">
          <widget class="QLineEdit" name="dispRoll"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>roll角度</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="dispYPos"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="dispYaw"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>yaw角速度</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>x坐标</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>y坐标</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="dispYawGyro"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>pitch角度</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="dispXPos"/>
         </item>
         <item row="6" column="0" colspan="2">
          <widget class="QLabel" name="dispFreq">
           <property name="text">
            <string>回传频率：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>yaw角度</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="dispPitch"/>
         </item>
        </layout>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="showGraph">
         <property name="text">
          <string>显示曲线</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="recordButton">
         <property name="text">
          <string>开始记录</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="setGroup">
     <property name="geometry">
      <rect>
       <x>368</x>
       <y>8</y>
       <width>341</width>
       <height>261</height>
      </rect>
     </property>
     <property name="title">
      <string>设置数据</string>
     </property>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>321</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QPushButton" name="setCaliGyro">
         <property name="text">
          <string>陀螺仪校准模式</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="setZero">
         <property name="text">
          <string>数据清零</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="setCaliEncoder">
         <property name="text">
          <string>码盘校准模式</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>进入校准后将会停止回传坐标，断电重启后恢复。校准码盘时，红灯会亮起，回传磁编码器的增量数据；校准陀螺仪时，绿灯会亮起，转发JY901S的数据，此时可以用维特智能上位机操作。</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>321</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>y坐标</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="setFreq">
         <property name="text">
          <string>设置频率</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="editYawPos"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>回传频率</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="setYaw">
         <property name="text">
          <string>设置yaw角度</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="editYPos"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>yaw角度</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="editXPos"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="setYPos">
         <property name="text">
          <string>设置y坐标</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>x坐标</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="setXPos">
         <property name="text">
          <string>设置x坐标</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="editSendFreq"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="graphGroup">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>631</width>
      <height>551</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QChartView" name="chartView" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>611</width>
       <height>471</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>9</y>
       <width>611</width>
       <height>54</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="1">
       <widget class="QCheckBox" name="autoScale">
        <property name="text">
         <string>自动缩放</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QCheckBox" name="drawYaw">
        <property name="text">
         <string>显示yaw角度</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QCheckBox" name="drawRoll">
        <property name="text">
         <string>显示roll角度</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>采样间隔(ms)          </string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QCheckBox" name="drawPtich">
        <property name="text">
         <string>显示pitch角度</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QCheckBox" name="drawYPos">
        <property name="text">
         <string>显示y坐标</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="drawTimeGap">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>100</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QCheckBox" name="drawYawGyro">
        <property name="text">
         <string>显示yaw角速度</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="drawXPos">
        <property name="text">
         <string>显示x坐标</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="autoScroll">
        <property name="text">
         <string>自动滚动</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pauseDraw">
        <property name="text">
         <string>暂停记录</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>QChartView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
